File: Arquetipo-Bancolombia-Heroe/domain/model/src/main/java/co/com/sofka/model/juego/Juego.java
package co.com.sofka.model.juego;

import co.com.sofka.model.jugador.Jugador;
import co.com.sofka.model.tablero.Tablero;
import lombok.*;

import java.util.List;

@Data
@ToString
@Builder(toBuilder = true)
@NoArgsConstructor
@AllArgsConstructor
public class Juego {

    private String id;
    private String ronda;
    private List<Jugador> jugadores;
    private Tablero tablero;
    private Jugador ganador;
}

File: frontend-hero/src/app/components/models/juego.ts

import { Jugador } from './jugador';
export class Juego {
    id?: string;
    ronda: string;
    jugadores: Jugador[];

    constructor(id: string, ronda: string, jugadores:Jugador[]){
        this.id = id;
        this.ronda = ronda;
        this.jugadores = jugadores;
    }
}